clients/da-vinci-client/src/main/java/com/linkedin/davinci/config/VeniceClusterConfig.java

```diff
@@ -93,6 +94,8 @@ public class VeniceClusterConfig {
   private final Map<String, SecurityProtocol> kafkaBootstrapUrlToSecurityProtocol;
   private final Optional<SSLConfig> sslConfig;
 
+  private static final String SEPARATE_TOPIC_SUFFIX = "_sep";
```

Comment:
nit: can reuse Utils's public static variable.

Reply:
This is no longer used so I am going to remove it